src/index.js

```diff
@@ -56,18 +56,27 @@ class Hr extends Component {
   }
 
   render() {
+    const { marginLeft, marginRight } = this.props;
+
     return (
-      <View style={{ flexDirection: 'row', alignItems: 'center', marginLeft: 8, marginRight: 8 }}>
-        {this.renderInner() }
-      </View>
+        <View style={{ flexDirection: 'row', alignItems: 'center', marginLeft: {marginLeft}, marginRight: {marginRight} }}>
```

Comment:
Shouldn't this be `<View style={{ flexDirection: 'row', alignItems: 'center', marginLeft, marginRight }}>`?